extensions/sql-migration/src/dialog/migrationCutover/migrationCutoverDialog.ts

```diff
@@ -150,54 +153,52 @@ export class MigrationCutoverDialog {
 			}).component();
 			flexFile.addItem(lastSSN.flexContainer, {
 				CSSStyles: {
-					'width': '200px'
+					'width': '250px'
```

Comment:
Would suggest making a single constant for these so you don't have to spend as much time updating them in the future.